packages/wrangler/src/__tests__/jest.setup.ts

```diff
@@ -118,3 +118,5 @@ jest.mock("xdg-app-paths", () => {
 		}),
 	};
 });
+
+jest.mock("create-cloudflare");
```

Comment:
Is it possible to not mock this, and have an actual integration test since the template project will be created in a temporary file system and removed afterward anyways?